core/resources/scene/scene.yaml

```diff
@@ -16,6 +16,22 @@ textures:
     water-texture:
         url: normals.jpg
         filtering: nearest
+
+# provide global fontfile names to be used for this stylesheet
+# these could be bundled with the tangram repo or could be loaded from system path
+# priority is given to bundled fonts
+fonts:
+    font1:
+        name: Roboto
+        weight: Regular
+    font2:
+        name: FireSans
+        weight: Medium
+    font3:
+        name: Roboto
```

Comment:
@bcamper what do you think of the suggested syntax revision above? I too prefer the map-style syntax because of it's suitability for rule inheritance.